Curtains/Curtains.Application/CurtainsService/EmailSender.cs

```diff
@@ -0,0 +1,40 @@
+using System.Net;
+using System.Net.Mail;
+using Curtains.Application.Interfaces;
+using Microsoft.Extensions.Configuration;
+
+namespace Curtains.Application.CurtainsService
+{
+    public class EmailSender : IMessageSender
+    {
+        private readonly IConfiguration _config;
+        public EmailSender(IConfiguration config)
+        {
+            _config = config;
+        }
+
+        /// <summary>
+        /// Sends a message to a specific address to an email address
+        /// </summary>
+        /// <param name="customerEmail">The mail to which the message is sent</param>
+        public async Task SendAsync(string subject, string body, string customerEmail)
+        {
+            var systemAddress = new MailAddress(_config["EmailSettings:SystemEmail"], _config["EmailSettings:SystemName"]);
```

Comment:
please move `"EmailSettings:SystemEmail"` into `const readonly` variable and use it

Reply:
other const values too